cypress/e2e/inventory/holdings/create-new-marc-holdings.cy.js

```diff
@@ -0,0 +1,94 @@
+import TopMenu from '../../../support/fragments/topMenu';
+import HoldingsRecordView from '../../../support/fragments/inventory/holdingsRecordView';
+import TestTypes from '../../../support/dictionary/testTypes';
+import QuickMarcEditor from '../../../support/fragments/quickMarcEditor';
+import InventoryInstance from '../../../support/fragments/inventory/inventoryInstance';
+import DevTeams from '../../../support/dictionary/devTeams';
+import DataImport from '../../../support/fragments/data_import/dataImport';
+import JobProfiles from '../../../support/fragments/data_import/job_profiles/jobProfiles';
+import Logs from '../../../support/fragments/data_import/logs/logs';
+import getRandomPostfix from '../../../support/utils/stringTools';
+import InventoryInstances from '../../../support/fragments/inventory/inventoryInstances';
+import Users from '../../../support/fragments/users/users';
+import Permissions from '../../../support/dictionary/permissions';
+import InventorySteps from '../../../support/fragments/inventory/inventorySteps';
+
+describe('Create holding records with MARC source', () => {
+  const marcFile = {
+    marc: 'oneMarcBib.mrc',
+    fileName: `testMarcFile.${getRandomPostfix()}.mrc`,
+    jobProfileToRun: 'Default - Create instance and SRS MARC Bib',
+    numOfRecords: 1,
+  };
+
+  const testData = {};
+  const recordIDs = [];
+
+  function getCurrentDateYYMMDD() {
+    const padWithZero = value => String(value).padStart(2, '0');
+    const initialCurrentDate = new Date();
+    return `${initialCurrentDate.getFullYear().toString().substring(2)}${padWithZero(initialCurrentDate.getMonth() + 1)}${padWithZero(initialCurrentDate.getDate())}`;
+  }
+
+  before(() => {
+    cy.createTempUser([
+      Permissions.inventoryAll.gui,
+      Permissions.uiQuickMarcQuickMarcHoldingsEditorCreate.gui,
+      Permissions.uiQuickMarcQuickMarcHoldingsEditorAll.gui,
+    ]).then(createdUserProperties => {
+      testData.userProperties = createdUserProperties;
+
+      cy.loginAsAdmin({ path: TopMenu.dataImportPath, waiter: DataImport.waitLoading }).then(() => {
+        DataImport.uploadFile(marcFile.marc, marcFile.fileName);
+        JobProfiles.waitLoadingList();
+        JobProfiles.searchJobProfileForImport(marcFile.jobProfileToRun);
+        JobProfiles.runImportFile();
+        JobProfiles.waitFileIsImported(marcFile.fileName);
+        Logs.checkStatusOfJobProfile('Completed');
+        Logs.openFileDetails(marcFile.fileName);
+        Logs.getCreatedItemsID().then(link => {
+          recordIDs.push(link.split('/')[5]);
+        });
+      });
+    });
+  });
+
+  beforeEach('Login', () => {
+    cy.login(testData.userProperties.username, testData.userProperties.password, { path: TopMenu.inventoryPath, waiter: InventoryInstances.waitContentLoading });
```

Comment:
Can we move it to 'before' hook if we have one test in the file

Reply:
The same. Fixed - moved to `before` hook